extensions/objects/models/objects.py

```diff
@@ -595,7 +595,7 @@ def normalize(cls, raw):
                 normalizing if necessary.
 
         Returns:
-            *. The normalized object containing the Graph schema.
+            dict. The normalized object containing the Graph schema.
```

Comment:
That's great; how about the remaining similar objects in this file?

Reply:
done